tools/ci-ts/test-helpers/chainlink-cli.ts

```diff
@@ -0,0 +1,42 @@
+import { ethers } from 'ethers'
+import execa from 'execa'
+import { JobSpec, JobRun } from '../../../operator_ui/@types/operator_ui'
+
+const API_CREDENTIALS_PATH = '/run/secrets/apicredentials'
+
+export async function login(): Promise<void> {
+  await execa('chainlink', ['admin', 'login', '--file', API_CREDENTIALS_PATH])
+}
+
+export async function getJobs(): Promise<JobSpec[]> {
+  const { stdout } = await execa('chainlink', ['-j', 'jobs', 'list'])
```

Comment:
We could make these functions easier to use in tests by making them synchronous. https://github.com/sindresorhus/execa#execasyncfile-arguments-options

Reply:
oh dang it's much cleaner now